tests/playwright-tests/README.md

```diff
@@ -0,0 +1,47 @@
+**How to install Playwright and set up VS Code?**
+
+# Here are the prerequisites to setting up VS Code:
+
+* nodejs : Download and Install Node JS from
+https://nodejs.org/en/download/
+
+* VS Code Editor: Download and Install VS code from 
+https://code.visualstudio.com/download
+* Playwright VS Code plugin
+* ES-lint extension
+* Playwright test runner
+
+
+# Installation
+
+Step 1: Navigate to 'Playwright-tests' folder
```

Comment:
Would suggest making use of Markdown's numbered list feature here.

Rather than needing to manually say "Step 1:", "Step 2:" etc, if you replace every "Step X:" with "1. " instead, so it would look like:

```
1. Navigate to 'Playwright-tests' folder
1. Ensure the NodeJS version is above 14 from the command prompt.  Blah blah.
    node -v
1. Launch the VS code
etc etc
```

Note all of these lines begin with "1. ", not "1. ", "2. ", "3. " etc etc.  This is so that you can slot in new items anywhere in the list but you don't have to spend ages updating the numbers.  By putting them all as "1"s, it automatically spits out a correctly numbered list.

Also, where we include script snippets in READMEs, we tend to put them in a code block to show it's a script or a command to run e.g:

node -v

would become:

~~~
```bash
node -v
```
~~~

which in the README, would then look like:

```bash
node -v
```